provisioning/azure-devops-deploy.sh

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
 set -Eeuo pipefail
 
+export COMMIT=$(echo "$BUILD_BUILDID" | cut -d'-' -f 2)
```

Comment:
IMHO by bylo lepší to rozdělit, `export` neumí probublat chyby

```
COMMIT=$(echo "$BUILD_BUILDID" | cut -d'-' -f 2)
export COMMIT
```

Reply:
👍